test/AutoRest.TestServer.Tests/LowLevel/DecoratorTests.cs

```diff
@@ -25,26 +24,5 @@ public class DecoratorTests
             Assert.IsNotNull(attribute);
             Assert.AreEqual("deprecated for test", ((ObsoleteAttribute)attribute).Message);
         }
-
```

Comment:
Why we remove this test?

Reply:
Because the models mockapi-related operations have been removed, there are no longer any operations handled with @deprecated(""). This test case has lost its object to test.

Reply:
Your justification is not correct. The object it is leveraging becomes unavailable is not the reason you could remove. You should understand what this test is for, before we decide whether it could be deleted, or move to another target.